python/tk_desktop/desktop_window.py

```diff
@@ -336,6 +342,17 @@ def _load_settings(self):
         # Update the project at the very end so the Python process is kicked off when everything
         # is initialized.
         project_id = self._settings_manager.retrieve("project_id", None, self._settings_manager.SCOPE_SITE)
+        if project_id == 0:
+            # 0 means will be displaying all of the projects
+            try:
+                from sgtk.util.metrics import EventMetric as EventMetric
+                EventMetric.log(EventMetric.GROUP_NAVIGATION,
+                                "Viewed Projects",
+                                context=sgtk.platform.current_bundle())
```

Comment:
Should probably use the `engine` here as well since this will always be a "Desktop" metric.

Reply:
Suggest `bundle` as mentioned in core PR.